framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -15,8 +15,58 @@ using namespace mapf;
 
 using namespace beerocks::bpl;
 
+#define RETURN_OK 0
```

Comment:
0 for success and -1 for error is the convention in the other BPL's. and since they are replaceable the convention must stay.
And I'd rather have one place to change those return values instead of going over each API...